packages/sdk-components-react/src/__generated__/vimeo.props.ts

```diff
@@ -41,7 +38,6 @@ export const props: Record<string, PropMeta> = {
   about: { required: false, control: "text", type: "string" },
   content: { required: false, control: "text", type: "string" },
   datatype: { required: false, control: "text", type: "string" },
-  inlist: { required: false, control: "text", type: "string" },
```

Comment:
also seems type string

Reply:
any in types

Reply:
I can't even find an attribute inlist anywhere in the spec, how did we get there, how can we get rid of it and maybe some similar cases?

Reply:
React.HTMLAttributes

Reply:
super weird, I can't find it in the spec at all

Reply:
we are already using ComponentProps<'div'> so we are probably already relying on React.HTMLAttributes

Reply:
Yes, this is what's inside of React.HTMLAttributes